tinker/templates/e-announcements/results.html

```diff
@@ -15,13 +15,21 @@
             </p>
             <div class="large-6 columns small-3">
                 <ul class="btn-group-horizontal">
+                {% set first_date = form.first_date.split(" ") -%}
                     {% if form.workflow_status %}
                         <li>This announcement is pending approval.</li>
                     {% else %}
                         {% if not form.first_date_past and not form.second_date_past %}
-                            <li>
-                                <a href="{{ url_for('EAnnouncementsView:edit', e_announcement_id=form.id) }}"
-                                   class="btn btn-primary">Edit</a></li>
+                                {% if (first_date[1] == tomorrow.month and first_date[2] != tomorrow.day and first_date[3] == tomorrow.year) and hour_today > 13 %}
```

Comment:
Ah, another part to this. Its 1pm not the day before. But rather 1pm of the business day before. Which means friday at 1pm is the cutoff for the monday morning e-annz.

Reply:
and holidays enter into play as well. I believe there is some code already in the repo for finding holidays.

Reply:
added in holidays logic